pkg/networkmanager/networking.css

```diff
@@ -268,21 +268,47 @@ h1 .onoff-ct {
     }
 }
 
-#zones-listing,
-#services-listing {
-    margin: 2rem 0;
+#zones-listing {
+    margin: 2rem 3rem;
 }
 
-#services-listing .listing-ct-body h5 {
-    margin: 1.5rem 0 1rem;
+#firewall-heading {
+    background: #FFFFFF;
+    padding: 10px;
```

Comment:
What if you set this to `padding: 1rem 3rem;` instead.
That will make it line up with the lists below.